tracer/src/Datadog.Trace/PDBs/SourceLink/AzureDevOpsSourceLinkUrlParser.cs

```diff
@@ -46,15 +53,15 @@ internal override bool TryParseSourceLinkUrl(Uri uri, [NotNullWhen(true)] out st
                 return false;
             }
 
-            var segments = uri.AbsolutePath.Split(new[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
+            var segments = uri.AbsolutePath.Split(new char[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Can we remove this split and the following check and do it only in `BuildRepositoryUrl(uri)` ?